app/src/main/java/com/eleks/mowid/ui/worker/QuotesWorkerManager.kt

```diff
@@ -0,0 +1,12 @@
+package com.eleks.mowid.ui.worker
+
+interface QuotesWorkerManager {
+
+    fun execute(options: Options)
+}
+
+enum class Options {
```

Comment:
I would suggest name ExecutionOption. First - it should be singular since you can pass only 1 option. Second - it is easier to import correct Option since "Options" is very general naming